public/javascripts/edit_device.js

```diff
@@ -1053,6 +1053,21 @@ anlixDocumentReady.add(function() {
     }
   });
 
+  $(document).on('click', '[id^=voip-phone-pots-ports-]', function(event) {
+    let parentTr = $(event.target).closest('tr');
+    if (parentTr.data('allow-multi-port')) {
+      return;
+    }
+    let ports = event.target.closest('[class=row]').children;
```

Comment:
Acho válido criar uma classe para identificar o CheckboxRow, para evitar que um bug seja criado se alguém adicionar uma `row` do bootstrap em algum outro lugar, o que é bem possível dada a versatilidade da classe `row`. No caso teria que incluir essa classe lá onde o `row` é usado no CheckboxRow, de forma que este tivesse as duas classes: `row` e `checkbox-row`, para que o seletor aqui possa ser mais específico:
```suggestion
    let ports = event.target.closest('[class=checkbox-row]').children;
```